cli/command/stack/remove.go

```diff
@@ -13,56 +14,63 @@ import (
 )
 
 type removeOptions struct {
-	namespace string
+	namespaces []string
 }
 
-func newRemoveCommand(dockerCli *command.DockerCli) *cobra.Command {
+func newRemoveCommand(dockerCli command.Cli) *cobra.Command {
 	var opts removeOptions
 
 	cmd := &cobra.Command{
 		Use:     "rm STACK",
 		Aliases: []string{"remove", "down"},
 		Short:   "Remove the stack",
```

Comment:
Can you change this to;

```go
Short:   "Remove one or more stacks",
```

to be consistent with other commands that accept multiple options